server/src/models/Appointment.js

```diff
@@ -45,4 +45,18 @@ const AppointmentSchema = new mongoose.Schema({
   finalAgencyPayment: Number
 });
 
-export default mongoose.model('Appointment', AppointmentSchema);
+appointmentSchema.statics.findForUser = function(user, query = {}) {
```

Comment:
Create methods to clean up your routes.

Here I created a method that gets appointments for a particular user